cmake/onnxruntime_webassembly.cmake

```diff
@@ -268,7 +268,10 @@ else()
   endif()
 
   if (onnxruntime_USE_WEBNN)
-   set_property(TARGET onnxruntime_webassembly APPEND_STRING PROPERTY LINK_FLAGS " --bind -sWASM_BIGINT")
+    set_property(TARGET onnxruntime_webassembly APPEND_STRING PROPERTY LINK_FLAGS " --bind -sWASM_BIGINT")
+    if (onnxruntime_DISABLE_RTTI)
+      set_property(TARGET onnxruntime_webassembly APPEND_STRING PROPERTY LINK_FLAGS " -fno-rtti -DEMSCRIPTEN_HAS_UNBOUND_TYPE_NAMES=0")
```

Comment:
Is this flag duplicated with setting in global flags for -fno-rtti ?

Reply:
I thought so, but the test will fail with error `out of memory bound`. It just magically passes if I add duplicated flags here.

Reply:
I guess that `out of memory bound` error is unrelated. It is a random failure only happened to onnxruntime_test_all with multi-threaded enabled. And this issue is being investigated by @snnn.

Reply:
I think his out of memory was not the CI error we know.
Anyway, I merge this one so I can restart CI for my PR to see if that all would be happy.